modules/weko-workflow/weko_workflow/utils.py

```diff
@@ -1525,3 +1528,93 @@ def update_indexes_public_state(item_id):
                 index.public_state = True
     if update_db:
         db.session.commit()
+
+
+def get_account_info(user_id):
```

Comment:
This function is duplicate with Line 1428: def get_account_info(user_id).

Reply:
Deleted it.